bin/preview-deploy/aws.sh

```diff
@@ -104,6 +104,7 @@ function createNewInstance() {
     --subnet-id $AWS_SUBNET \
     --tag-specification "ResourceType=instance,Tags=[{Key=Name,Value=eAPD PR $PR_NUM},{Key=environment,Value=preview},{Key=github-pr,Value=${PR_NUM}}]" \
     --user-data file://aws.user-data.sh \
+    --key-name eapd_bbrooks \
```

Comment:
did you mean to have this in there?

Reply:
Yes, I need it to address the log locations in Staging. Thank you for calling it out because it does need to be removed. It's not actual key material, so if it sneaks into prod it isn't the worst thing, but I don't want to leave it in for sure.